internal/pkg/daemon/bpfrecorder/bpfrecorder.go

```diff
@@ -546,6 +553,8 @@ func toStringByte(array []byte) string {
 
 func (b *BpfRecorder) processEvents(events chan []byte) {
 	b.logger.Info("Processing events")
+	// Update mntns to system_mntns
+	b.updateSystemMntnsMap()
```

Comment:
Let's do this directly on init via rodata.

Reply:
Is there any way to pass parameters before the ebpf program starts? From libbpf go,I didn't find some related function,such as bpf.NewModuleFromBufferArgs(*args).

Reply:
Ah looks like there is no support for it yet: https://github.com/aquasecurity/libbpfgo/issues/27

Therefore we would have to use a map. But one with just a single entry to reduce the memory usage.